src/@types/event.ts

```diff
@@ -180,26 +180,32 @@ export const UNSTABLE_ELEMENT_FUNCTIONAL_USERS = new UnstableValue(
 /**
  * A rel_type that affects visibility of a message,
  * as per https://github.com/matrix-org/matrix-doc/pull/3531
+ *
+ * @experimental
  */
-export const UNSTABLE_MSC3531_VISIBILITY_CHANGE_REL_TYPE = new UnstableValue(
+export const MSC3531_VISIBILITY_CHANGE_REL_TYPE = new UnstableValue(
```

Comment:
sorry, also meant to say that the MSC number should be dropped from the constant's name as well. Also, `EVENT` should be somewhere in there to line up with all the other EventVisibility things.